test/Header/SetCookieTest.php

```diff
@@ -343,6 +343,12 @@ public function testDoesNotAcceptCookieNameFromArbitraryLocationInHeaderValue()
         $this->assertNotEquals('leo_auth_token', $setCookieHeader->getName());
     }
 
+    public function testEmptyExpiresSection()
+    {
+        $cookie = 'Set-Cookie:path=/; Expires=; HttpOnly;';
+        $setCookieHeader = SetCookie::fromString($cookie);
```

Comment:
> you can use `@expectedException`

If you expect an exception you can use that, but in this case you expecting the opposite: no exception. You can not use annotations for that, because there is none for "not expecting an exception".

> Tests should not always have assertions, if test has passed without exceptions than it is ok, pass or fail

I agree it is not mandatory, but IMO it is better to be explicit when testing.